gpflow/optimizers/scipy.py

```diff
@@ -7,16 +7,16 @@
 
 __all__ = ["Scipy"]
 
-Variables = Iterable[tf.Variable]
-StepCallback = Callable[[int, Variables, List[tf.Tensor]], None]
+Variables = Iterable[tf.Variable]  # deprecated
```

Comment:
I'm sorry for being late with a reply. Why is that deprecated instead of replacing `Iterable` with `Sequence`. You could still reuse `Variables` alias in the code and minimize the change. I think it is a sensible thing to do, could you please leave `Variables` alias in the code below (where Sequence[tf.Variable] is used). Thanks!

Reply:
i'm also not in favour of changing it now. i've already reviewed it weeks ago and don't want to have to look at it again.